content/guides/production-build.md

```diff
@@ -71,8 +71,87 @@ module.exports = {
   /*...*/
 };
 ```
+When we do have multiple configurations in mind for different environments, the easiest way is to write seperate js files for 
+each environment. For example:
+##config/dev.js
+```js
+module.exports = function(env) {
+    debug: true,
+    devtool: 'cheap-module-source-map',
+    output: {
+      path: path.join(__dirname, '/../dist/assets'),
+      filename: '[name].bundle.js',
+      publicPath: publicPath,
+      sourceMapFilename: '[name].map'
+    },
+    devServer: {
+      port: 7777,
+      host: 'localhost',
+      historyApiFallback: true,
+      noInfo: false,
+      stats: 'minimal',
+      publicPath: publicPath
+    },
+ });
+```
+##config/prod.js
+```js
+module.exports = function(env) {
+debug: false,
+    output: {
+      path: path.join(__dirname, '/../dist/assets'),
+      filename: '[name].bundle.js',
+      publicPath: publicPath,
+      sourceMapFilename: '[name].map'
+    },
+    plugins: [
+      new DedupePlugin(),
```

Comment:
Thanks for pointing out. Removed it. The idea was just to provide a sample config and the example was straight out of my code, which am still updating.